portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,11 +26,23 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  
+  private ArrayList<String> messages = new ArrayList<String>();
+
+  @Override
+  public void init() {
+    messages.add("Hello, how are you?");
+    messages.add("Hello, what is your name?");
+    messages.add("Hello, what is your age?");
+  }
   @Override
```

Comment:
Please add a space above this method.
ref: https://google.github.io/styleguide/javaguide.html

Reply:
I am confused about what you mean, by adding a space?

Reply:
I think I did it

Reply:
Yes. Thank you.